pulpcore/app/serializers/repository.py

```
@@ -246,11 +246,14 @@ class Meta:
 
 class RepositoryVersionCreateSerializer(ModelSerializer, NestedHyperlinkedModelSerializer):
     add_content_units = serializers.ListField(
-        help_text=_('A list of content units to add to a new repository version'),
+        help_text=_('A list of content units to add to a new repository version. This content is '
+                    'added after remove_content_units are removed.'),
         write_only=True
     )
     remove_content_units = serializers.ListField(
-        help_text=_('A list of content units to remove from the latest repository version'),
+        help_text=_("A list of content units to remove from the latest repository version. "
```

I imagined that `['*']` would be valid but `['someurlhere/UUID', '*']` to not be. It's not a big deal but since all other entries are ignored perhaps this should fail hard? what do you think?

---

Done. Can you check out the change?